spec/spec_helper.rb

```diff
@@ -23,6 +25,11 @@
 
   config.shared_context_metadata_behavior = :apply_to_host_groups
 
+  config.before(:each) do
```

Comment:
Are we actually using this anywhere? 🤔  I'd rather we stub the `MojFileUploader` instead of the URL if we can...